components/customer-dashboard.tsx

```diff
@@ -73,6 +78,16 @@ const CustomerDashboard = () => {
     enabled: !!customerStore?.customer?.id,
   });
 
+  const getMintStatus = async (address: Address) => {
```

Comment:
why are we not using the useReadContract hook here?

Reply:
becuase i need to get mintstatus for each point, so can't use that in loop